common/model/transaction.h

```diff
@@ -85,106 +92,262 @@ struct _iota_transaction {
   int64_t attachment_timestamp_upper;
   // 27 trytes = 81 trits
   flex_trit_t nonce[FLEX_TRIT_SIZE_81];
+  // 27 trytes = 81 trits
+  flex_trit_t tag[FLEX_TRIT_SIZE_81];
+};
+
+struct _iota_transaction_fields_consensus {
   // 81 trytes = 243 trits
   flex_trit_t hash[FLEX_TRIT_SIZE_243];
-  // Total 2754 trytes
+  // 81 trytes = 243 trits
+  flex_trit_t bundle[FLEX_TRIT_SIZE_243];
```

Comment:
let's move the bundle back to the essence for now?

Reply:
done